internal/host/transition.go

```diff
@@ -59,7 +59,7 @@ func (th *transitionHandler) PostRegisterHost(sw stateswitch.StateSwitch, args s
 		// so we reset the hw info and progress, and start the discovery process again.
 		if host, err := updateHostProgress(params.ctx, log, th.db, th.eventsHandler, sHost.host.ClusterID, *sHost.host.ID, sHost.srcState,
 			swag.StringValue(sHost.host.Status), statusInfoDiscovering, sHost.host.Progress.CurrentStage, "", "",
-			"inventory", "", "discovery_agent_version", params.discoveryAgentVersion, "bootstrap", false); err != nil {
+			"inventory", "", "discovery_agent_version", params.discoveryAgentVersion, "bootstrap", false, "ntp_sources", ""); err != nil {
```

Comment:
maybe need to add to `PostEnableHost` as well

Reply:
Interesting. What about all of those parameters? Should they get resetted as well?

Reply:
if not then it's a bug. 
In this specific case there is a chance that host will become known bug maybe the ntp was already changed while the host was disabled. 
It's not a big issue because he will get the new configuration in the next iteration of next step but still.